arm-web/2016-06-01/swagger/connectionGateways.json

```diff
@@ -39,7 +39,10 @@
         "tags": [
           "Gateways"
         ],
-        "operationId": "Gateways_ListBySubscription",
+        "x-ms-examples": {
+          "Task get": { "$ref": "../examples/ConnectionGatewaysGet.json" }
```

Comment:
this reference doesn't point to any file. Please fix. I think you mean to refer to "ConnectionGatewayGet.json". This is causing CI to fail.
I'd strongly recommend running the linter and model validation tools locally before submitting the PR, see https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist